pkg/macos/pkg-scripts/postinstall

```diff
@@ -77,13 +77,65 @@ fi
 # This is a special tool to make it easier for the user to get started setting
 # up salt
 log "Symlink: Creating symlink for salt-config..."
-ln -sf "$BIN_DIR/salt-config.sh" "$SBIN_DIR/salt-config"
+ln -sf "$INSTALL_DIR/salt-config" "$SBIN_DIR/salt-config"
```

Comment:
This one is still a bash script. It's not a binary created by salt. I wouldn't change this one. Unless it is now in `$INSTALL_DIR` instead of `$BIN_DIR`. But it's still a shell script.

Reply:
Fixed.